src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22452.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Diagnostics;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue22452 : _IssuesUITest
+{
+	public Issue22452(TestDevice device) : base(device) { }
+
+	public override string Issue => "Fix error when running new template maui app on iOS";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void NavigationBetweenFlyoutItems()
+	{
+		App.WaitForElement("FlyoutItem1");
```

Comment:
aw dang

unfortunately this automationid's don't work on WinUi it looks like

`NavigationBetweenFlyoutItems` is failing on windows. 

You can add a templated flyout item and then just place a label inside of it and use that?